autohdr.sh

```diff
@@ -16,12 +16,17 @@ echo "Processing directory $DIR"
 OUTPUT_FILE=$(ls $DIR/*.JPG | head -n 1 | sed -e 's/\.JPG$/_hdr.pto/')
 echo "creating '$OUTPUT_FILE'..."
 
-pto_gen $DIR/*.JPG -o $OUTPUT_FILE
+NUM_IMAGES=$(ls $DIR/*.JPG -l | wc -l)
+
+
+pto_gen $DIR/*.JPG -s $NUM_IMAGES -o $OUTPUT_FILE
 cpfind -o $OUTPUT_FILE $OUTPUT_FILE 
 cpclean -o $OUTPUT_FILE $OUTPUT_FILE 
 autooptimiser -p -o $OUTPUT_FILE $OUTPUT_FILE 
 
+
 # -p 0 => rectaliniar projection
-pano_modify -o $OUTPUT_FILE -p 0 --ldr-file=JPG --output-type=BF $OUTPUT_FILE
+# --output-type=FB => fuse then blend, it shouldn't really matter which one we do first for an HDR but my expereince is FB helps avoid the "enblend: excessive overlap detected" error
+pano_modify -o $OUTPUT_FILE -p 0 --ldr-file=JPG --output-type=FB $OUTPUT_FILE
```

Comment:
It is fine with me to revert it.

Interesting, I just read about color banding on some forums. Especially if the "--fine-mask" option works to resolve the "enblend: excessive overlap detected" error like you suggested